### ISSUE TITLE
Start-of-new-day settings not applying correctly

### ISSUE THREAD

**Issue by ps-goog** [ankidroid/Anki-Android#5082]
###### Reproduction Steps
Still doesn't run correctly changing app's language so I can't show English version

Screenshots

###### Expected Result
After return form settings main widow should be redrawn

###### Actual Result
Main widow isn't redrawn
![screenshot_20181027-173122_ankidroid](https://user-images.githubusercontent.com/43900825/47606137-1fa46700-da10-11e8-9361-578dc507953c.jpg)
![screenshot_20181027-173014_ankidroid](https://user-images.githubusercontent.com/43900825/47606139-292dcf00-da10-11e8-9378-b57e108d289e.jpg)
![20181027_174041](https://user-images.githubusercontent.com/43900825/47606142-31860a00-da10-11e8-964a-2569d4c8e001.jpg)

###### Debug info
AnkiDroid Version = 2.9alpha47
Android Version = 8.0.0
ACRA UUID = dd27ecef-281f-411a-b121-c44b1ca1bdb7

###### Research
*Enter an [ x ] character to confirm the points below:*

[  ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[  ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[  ] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
Definitely an issues but duplicates #4729

**Comment by mikehardy** [comment]
I hope to fix this before 2.9 but don't have my PR working yet unfortunately

**Comment by ps-goog** [comment]
The main issue is about to not redraw information about deck.
Language only for information why I attach Polish screenshots.

**Comment by mikehardy** [comment]
Ah okay. I can't look closely this moment but I'll look later. Thanks!

**Comment by mikehardy** [comment]
What are the exact steps to reproduce? Can you please follow the bug reporting template? Because this really does look like a duplicate of #4729 now that I'm looking closer. Start AnkiDroid in one language (or system language), switch to another language, GUI is confused and shows some elements in one language and some in another? That's #4729...

**Comment by ps-goog** [comment]
1. Set [begining of new day] to for example 5 pm
2. Finish learning process -  all deck have got zeros in learning card
3. wait to the next day, for example 2 am
4. Go to settings and change [begining of new day] to 1am, back and
5. You see all zeros instead of seeing information about learning card in 'next day'

**Comment by mikehardy** [comment]
Did you sync up after step 2? I would have but that triggers an unrelated issue about this setting being overwritten by the server - you might check to see if the configured time is still as expected

**Comment by ps-goog** [comment]
You shouldn't sync, even though you see zeros you can push button [learn now] twice and it's possible to start learning process and usually after back to main screen all looks well.

**Comment by mikehardy** [comment]
Very interesting - I'm actually happy to hear you didn't sync so this reproduction should be easy, even though you should sync of course for data safety ;-). Thanks for the report and the steps to reproduce

**Comment by ps-goog** [comment]
Of course I sync, even several time a day and send collection to Google drive.

Solution is very simple, redraw main window after returning from settings :-) I know that programming for Android isn't simple in complex application but everything can be fix.

**Comment by HairyPhil** [comment]
I am experiencing the same problem. My "start of next day" is set to "0 hours past midnight", but it triggers at 23:00. I am in France, on Central European Standard Time (UTC+1), so it is triggering at 22:00 UTC. AnkiDroid v2.8.4 on Android 9.0 on Pixel 3.

**Comment by HairyPhil** [comment]
Duplicate of #5637.

**Comment by david-allison-1** [comment]
Thanks for the link!
